src/components/Challenges/Challenge.query.tsx

```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag';
```

Comment:
This file was replaced with Step.query.tsx a few commits ago. I think you probably made a mistake while rebasing. It needs to be removed. If you made any changes here, you probably meant for them to be in Step.query.tsx